infomapcog/distances.py

```diff
@@ -73,32 +73,57 @@ def needleman_wunsch(x, y, lodict={}, gop=-2.5, gep=-1.75):
     'lodict' must be a dictionary with all symbol pairs as keys and
     match scores as values, or a False value (including an empty
     dictionary) to denote (-1, 1) scores. gop and gep are gap
-    penalties for opening/extending a gap.
+    penalties for opening/extending a gap; alternatively, you can set
+    'gop' to None and provide element/gap alignment costs.
 
     Returns the alignment score and one optimal alignment.
+    
+    >>> needleman_wunsch("AAAAABBBB", "AACAABBCB")
+    (5.0, [('A', 'A'), ('A', 'A'), ('A', 'C'), ('A', 'A'), ('A', 'A'), ('B', 'B'), ('B', 'B'), ('B', 'C'), ('B', 'B')])
+    >>> needleman_wunsch("banana", "mancala", local=True)
```

Comment:
I actually wanted to do something like it, but it was not immediately obvious how to do it, and for the test case I had returning just the alignment seemed sufficient.